apps/bluetooth/app.js

```diff
@@ -425,8 +425,8 @@ module.exports = function (activity) {
         }
         break
       case protocol.CONNECTION_STATE.DISCONNECTED:
-        if (lastIntent === 'bluetooth_disconnect') {
-          logger.debug('Suppress "disconnected" prompt while close.')
+        if (lastIntent !== 'disconnect_devices' && lastIntent !== 'disconnect_phone' && lastIntent !== 'disconnect_speaker') {
```

Comment:
Multiple conditions could be grouped into a simple one.
```suggestion
        if (lastIntent !== 'disconnect_devices' && lastIntent !== 'disconnect_phone' && lastIntent !== 'disconnect_speaker') {
```

Reply:
This looks no different. Could you tell me what's the benefit of converting multiple conditions to `Array.indexOf`?

Reply:
Yes, there is no difference between these two pattern. Though the later one would be easier to read IMO.

Reply:
> Yes, there is no difference between these two pattern. Though the later one would be easier to read IMO.

replaced by `startsWith`